shapeshift/web
# fix: dynamic text scaling

## Description

* fixes #850 – current `FiatInput` component still had same clipping issues on smaller screens
* Introducing basic dynamically resizable, reusable component `FlexibleInputContainer`

https://www.loom.com/share/934d4d4838d74148853d2baae7feb763

Note: Solution is still JS based, just reduced in complexity. CSS only solution would be quite challenging to pull off. For accurate scaling font size should depend on the text length which is accessible via JS only.

Good next step would be to make token input fields scalable as well.

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Testing

Please outline all testing steps

1. Pull branch locally and run `yarn` to install new deps, `cp .sample.env .env`, `yarn dev`
2. Type in a long number into the fiat currency input field.
3. Check how it works on different screen sizes.
